include/circt/Dialect/LTL/LTLOps.td

```diff
@@ -160,10 +160,7 @@ def ConcatOp : LTLOp<"concat", [Pure]> {
   }];
 }
 
-def RepeatOp : LTLOp<"repeat", [
-  Pure,
-  PredOpTrait<"'base' must be >= 0", CPred<"$base >= 0">>
```

Comment:
Why is this trait removed?

Reply:
it was generating a lot of warnings because the generated code is >= 0 by default

Reply:
Please feel free to push unrelated change separately to main.